src/Microsoft.ApiDesignGuidelines.Analyzers/Core/PassSystemUriObjectsInsteadOfStrings.cs

```diff
@@ -24,15 +29,201 @@ public abstract class PassSystemUriObjectsInsteadOfStringsAnalyzer : DiagnosticA
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Usage,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/ms182360.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            // this is stateless analyzer, can run concurrently
+            analysisContext.EnableConcurrentExecution();
+
+            // this has no meaning on running on generated code which user can't control
+            analysisContext.ConfigureGeneratedCodeAnalysis(GeneratedCodeAnalysisFlags.None);
```

Comment:
Ah, I didn't know these were available. Let me open a tracking item to go through all analyzers to enable both these features. Thanks!